tests/parser.cpp

```diff
@@ -277,11 +281,36 @@ TEST(Parser, positional_param_attachpoint)
   int: 1
 )PROG");
 
-  test_parse_failure(bpftrace, R"PROG(uprobe:$1a" { 1 })PROG");
-  test_parse_failure(bpftrace, R"PROG(uprobe:$a" { 1 })PROG");
-  test_parse_failure(bpftrace, R"PROG(uprobe:$-1" { 1 })PROG");
+  // TODO what are these tests meant to be checking for?
```

Comment:
These tests confused me. Can anyone help to explain why there are trailing, unmatched quotes (`"`) in each of them?

Do they serve a purpose or could they be a mistake?

Reply:
I git-blamed it and it looks like I added it. Probably a typo. Good that you're adding the expected error messages :)

Reply:
Cool, removed the trailing quotes!